static/js/cf-frontend.js

```diff
@@ -437,6 +459,8 @@ setFronendInfo(["Server error! Your code might be taking too much time to run or
 
   $("#executeBtn").attr('disabled', false);
   $("#executeBtn").click(executeCodeFromScratch);
+  $("#versionNumber").attr('disabled', false);
+  $("#versionNumber").click(executeVersion);
```

Comment:
Should the button be disabled once it has been pressed once? That is, at the end of `executeVersion` you disable the button to not allow running it again. Or is the button completely replaced by the version text and there is no way to press a button anymore?

Reply:
I updated it now so that the button itself doesn't change, it create a text box beside the button for the version number